requirements.txt

```diff
@@ -1,64 +1,60 @@
-aiogrpc==1.8
-aiohttp==3.8.4
+aiohttp==3.8.5
```

Comment:
Do you already use a `requirments.in` file? If not, let me know and I can give you some resources.